clusterDeployer.py

```diff
@@ -553,11 +554,13 @@ def wait_for_workers(self) -> None:
         lh = host.LocalHost()
         bf_workers = [x for x in self._cc.workers if x.kind == "bf"]
         connections: Dict[str, host.Host] = {}
-        while True:
+        for try_count in itertools.count(0):
             workers = [w.name for w in self._cc.workers]
-            if all(self.client().is_ready(w) for w in workers):
+            ready_workers = (self.client().is_ready(w) for w in workers)
+            if all(ready_workers):
                 break
 
+            logger.info(f"Not all workers ready (try #{try_count}). {len([r for r in ready_workers if r])} are not ready yet.")
```

Comment:
It lacks a `not` otherwise it count the ready workes.

Any maybe `sum(1 for r in ready_workers if not r)`

Reply:
Ah, right, thanks!

Reply:
We only need the number of nodes that are not ready.  I changed the code to compute that once, instead.